lldb/test/API/lang/swift/macro/macro.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import _SwiftSyntaxMacros
+
+public struct StringifyMacro: ExpressionMacro {
+  public static func expansion(
+    of macro: MacroExpansionExprSyntax, in context: inout MacroExpansionContext
+  ) -> ExprSyntax {
```

Comment:
We've been tweaking the signature of macros a bit, and this will need to be:

```suggestion
  public static func expansion(
    of macro: some FreestandingMacroExpansionSyntax, 
    in context: some MacroExpansionContext
  ) -> ExprSyntax {
```